mycity/mycity/intents/speech_constants/get_alerts_intent.py

```diff
@@ -4,3 +4,4 @@
 """
 
 NO_ALERTS = "There are no alerts. City services are running on normal schedules."
+NO_INCLEMENT_WEATHER_ALERTS = "There are currently no weather related alerts"
```

Comment:
Nitpick: `"There are no weather related alerts."` scans better for me. Ignore if you want